server/rosetta/client_online.go

```diff
@@ -481,13 +484,39 @@ func (c *Client) blockTxs(ctx context.Context, height *int64) (crgtypes.BlockTra
 
 func (c *Client) getHeight(ctx context.Context, height *int64) (realHeight *int64, err error) {
 	if height != nil && *height == -1 {
-		genesis, err := c.tmRPC.Genesis(ctx)
+		genesisChunk, err := c.tmRPC.GenesisChunked(ctx, 0)
```

Comment:
@fkneeland-figure  could we possibly highlight what you changed vs upstream

Reply:
I believe it was only changes to the go.mod file for rosetta and golang-lru versions?

Reply:
<img width="1319" alt="Screen Shot 2021-12-09 at 9 02 13 PM" src="https://user-images.githubusercontent.com/1173553/145515350-d4f8250d-df4c-49ca-8b4d-b22a162ba4de.png">
not sure about the LRU change, but these are the 4 files i see change, seems like an empty string added to server.go, is the only change, i wonder if we should PR against the [`v0.43.`](https://github.com/provenance-io/cosmos-sdk/tree/v0.44.3-pio) branch and then create a reales 0.44-pio.x branch for release? (the diff between `v0.44.3-pio` and this branch is super easy to follow

Reply:
i guess this https://github.com/provenance-io/cosmos-sdk/compare/provenance-io:release/v0.44.x...provenance-io:v0.44.3-pio plus @fkneeland-figure change of 1 file adds up 🤷